### PR TITLE
docs: add tip for skip_render issue

### PR DESCRIPTION
**Pull request by FoliageOwO**

### CODE REVIEW

File: docs/en/guide/index.md [hunk 1]
```diff
@@ -108,6 +108,19 @@ You can create a new `load-aplayer.js` file under the `source/js` folder in the
 
 > If the `js` folder does not exist, please create a new one.
 
+::: danger
+Hexo may put the folder of `source` directory as **renderable page**, this will cause files be rendered as `HTML`.
+
+To solve this issue, you also need to configure `skip render` for all `js` file in `config.yml`:
```
Comment (YunYouJun) [inline docs/en/guide/index.md]: `config.yml` should be `_config.yml`.

File: docs/guide/index.md [hunk 2]
```diff
@@ -160,6 +160,19 @@ npm i hexo-theme-yun@latest
 
 > 若 `js` 文件夹不存在，请新建。
 
+::: danger
+Hexo 会把 `source` 目录下的文件夹视为**可渲染的页面**，导致文件加载时会被渲染为 `HTML`。
+
+为了避免此问题，你还需要在 `config.yml` 为所有可能出问题的 `js` 文件配置跳过渲染：
```
Comment (YunYouJun) [inline docs/guide/index.md]: Same as above.

File: docs/en/guide/index.md (hunk 1 above)
Comment (FoliageOwO) [reply docs/en/guide/index.md]: solved

**Comment by YunYouJun** [comment]
Thanks!

**Pull request merged**